build_library/grub.cfg

```diff
@@ -69,6 +69,9 @@ fi
 if [ -f "($root)/flatcar/first_boot" ]; then
     set first_boot="flatcar.first_boot=detected"
 fi
+if [ -f "($root)/coreos/first_boot" ]; then
+    set first_boot="coreos.first_boot=detected"
```

Comment:
Why set coreos.first_boot instead of flatcar.first_boot?

Reply:
On one hand because it allows us to test this parameter and on the other because we could in the future give a deprecation warning for the old flag file location.

Reply:
The flag file location is separate from the grub command line parameter. I don't think it's good to add now a command line parameter that we don't intend to support.

Reply:
We indent to support for now it because we aim to be a drop-in replacement.

Reply:
But sure, I can remove it here as it's an internal thing.